great_expectations/checkpoint/util.py

```diff
@@ -227,6 +232,10 @@ def get_runtime_batch_request(
                 f'BatchRequest attribute "{key}" was specified in both validation and top-level CheckpointConfig.'
             )
     runtime_batch_request_dict.update(runtime_config_batch_request)
+    if batch_data is not None:
+        if "runtime_parameters" not in runtime_batch_request_dict:
+            runtime_batch_request_dict["runtime_parameters"] = dict()
```

Comment:
@roblim Minor -- but if you can, please change to `{}` instead of `dict()` -- this has been made our prevailing style.  Thank you!

Reply:
Addressed in b69f181ea44fc1bdb6af596c41328b514e7cca31